kasa/feature.py

```diff
@@ -139,7 +139,7 @@ class Category(Enum):
     #: Name of the property that allows accessing the value
     attribute_getter: str | Callable | None = None
     #: Name of the method that allows changing the value
```

Comment:
Docstring needs to be updated accordingly, but perhaps this change (given how it changes our "core") should go in in its own PR?

Reply:
Sure. You happy for me to just do that and merge?

Reply:
Sure, hmm, the `attribute_getter` has also a stale docstring, perhaps fix that also while you are at it? I'm also wondering if we should consolidate both to have the same type, what do you think?

Reply:
The getter is a sync callable but the setter expects an `async` method so not sure we want to consolidate the types.

Reply:
https://github.com/python-kasa/python-kasa/pull/1272